### DISCUSSION TITLE
Help Running TGV-QSM on Cluster

### DISCUSSION THREAD

**Discussion by FrancescaPentimalli** [neurodesk/neurodesk.github.io-hugo#464] [Q&A] [answered]
Hi Neurodesk Team,

I’m trying to run TGV-QSM on my university's HPC cluster. I followed the instructions to use run_transparent_singularity.sh, but I ran into this error:

**INFO:    Converting SIF file to temporary sandbox...
FATAL:   while extracting tgvqsm_1.0.0_20210629.simg: root filesystem extraction failed: extract command failed: ERROR  : Failed to create user namespace: user namespace disabled**
It seems this might be due to user namespace or fakeroot permissions not being enabled on the cluster. I’ve contacted IT support to address this, but I want to confirm:
Am I following the correct steps to run TGV-QSM with Neurodesk on a cluster?
Is enabling these permissions the only solution, or is there something else I can try?
Thanks for your help! :)

**Comment by stebo85** [comment, marked as answer]
It could be a wrong configuration in your cluster for apptainer/singularity. 

You could try our latest QSMxt container: https://qsmxt.github.io/QSMxT/

It's build with newer apptainer versions and might work for you. QSMxt includes TGV qsm and a lot of great wrappers to process the data quickly.
